generators/app/index.js

```diff
@@ -164,7 +211,11 @@ module.exports = class extends BaseGenerator {
         this.components = this.props.components;
         this.entities = this.props.entities;
         this.pollingTimeout = this.props.pollingTimeout;
-
+        // variable from previous states
+        /* //`${resourceDir}config/application.yml`
+        //`${testResourceDir}config/application.yml`
+        this.previousData = jsYaml.load(fsModule.readFileSync(`${resourceDir}config/application.yml`, 'utf8'));
+        this.log(this.previousData); */
```

Comment:
Don't forget to remove those comment lines before to merge the PR

Reply:
Yes, of course, it's still in progress :)